gson/src/main/java/com/google/gson/FormattingStyle.java

```diff
@@ -44,35 +51,35 @@ private FormattingStyle(String newline, String indent) {
     }
     if (!indent.matches("[ \t]*")) {
       throw new IllegalArgumentException(
-          "Only combinations of spaces and tabs allowed in indent.");
+          "Only combinations of spaces and tabs are allowed in indent.");
     }
     this.newline = newline;
     this.indent = indent;
   }
 
   /**
-   * Creates a {@link FormattingStyle} with the specified newline setting.
+   * Creates a {@code FormattingStyle} with the specified newline setting.
```

Comment:
Have changed this to `@code` because linking to the class declaring this method is probably not necessary.

The (somewhat dated) ["How to Write Doc Comments for the Javadoc Tool" guide](https://www.oracle.com/de/technical-resources/articles/java/javadoc-tool.html) also says: "Use in-line links economically"

Reply:
The point about validating the referenced name makes sense, but to me that appears to be rather a workaround for missing Javadoc functionality (e.g. `@code` tag which represents a reference).

Personally I still think in the HTML output the links can be distracting / confusing, giving the impression that they refer to a different, similarly named class.

I have reverted these `@link` changes now though.